util/src/main/java/io/kubernetes/client/Copy.java

```diff
@@ -150,14 +150,15 @@ public void copyDirectoryFromPod(
         }
       }
     }
-    try {
+    // No need to call waitFor() as proc is closed on exit(stream == 3).
+    /*try {
```

Comment:
Please just delete code instead of commenting it out.

Reply:
review comments are incorporated.